code/modules/clothing/suits/wintercoats.dm

```diff
@@ -215,8 +216,15 @@
 	desc = "A lab-grade winter coat made with acid resistant polymers. For the enterprising chemist who was exiled to a frozen wasteland on the go."
 	icon_state = "coatchemistry"
 	inhand_icon_state = "coatchemistry"
+/obj/item/clothing/suit/hooded/wintercoat/medical/chemistry/Initialize(mapload)
+	. = ..()
+	allowed += list(
+		/obj/item/storage/bag/chemistry,
+	)
```

Comment:
```suggestion
	allowed += /obj/item/storage/bag/chemistry
```

Reply:
Will change and push them once I get connection back